include/retdec/fileformat/types/import_table/import_table.h

```diff
@@ -57,7 +60,7 @@ class ImportTable
 		/// @{
 		void computeHashes();
 		void clear();
-		void addLibrary(std::string name);
+		void addLibrary(std::string name, bool bIsMissingDependency = false);
```

Comment:
We are not using hungarian notation anywhere in our projects so it seems rather inconsistent. Could you please rename it to `isMissingDependency`?